### PR TITLE
Add source for KAEV Niederlausitz

### PR DESCRIPTION
**Pull request by aschobba**
Adds source for KAEV Niederlausitz.

https://www.kaev.de/Info-und-Service/Tourenplan/Tourenplan-Abfalltermine.html

### CODE REVIEW

File: doc/source/kaev_niederlausitz.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Kommunaler Abfallentsorgungsverband Niederlausitz
+
+Support for schedules provided by [kaev.de](https://www.kaev.de/).
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: bsr_de
```
Comment (mampfes) [inline doc/source/kaev_niederlausitz.md:10]: Wrong name. Copy-paste error?

File: doc/source/kaev_niederlausitz.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      args:
+        abf_strasse: STRASSE
```
Comment (mampfes) [inline doc/source/kaev_niederlausitz.md:12]: I would change the order to Ort -> Ortsteil -> Strasse (here and in all other places in the document).

File: doc/source/kaev_niederlausitz.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        abf_ort: "Luckau"
+		abf_ot: "Luckau"
+```
+
+
+## How to get the source arguments
+
+
+1. Go to your calendar at `https://www.kaev.de/Info-und-Service/Tourenplan/Tourenplan-Abfalltermine.html`.
+2. Enter your location.
```
Comment (mampfes) [inline doc/source/kaev_niederlausitz.md:55]: I followed the link and entered "Staakow" in the search field. The site returned results for that. But how do I get the values for the 3 fields from that?

Would it be easier to use just one configuration option for the source which is exactly the content of the search field?

**Review by mampfes — CHANGES_REQUESTED** [review]
Please add the source to README.md and info.md in the top level folder.

File: doc/source/kaev_niederlausitz.md (hunk 3 above)
Comment (aschobba) [reply doc/source/kaev_niederlausitz.md:55]: So mean remove abf_ort, abf_ot and abf_strasse and just an abf_search where the user has to specify the full string from search? Currently, it's more like you enter everything you know and then the script is trying to find the right option.

File: doc/source/kaev_niederlausitz.md (hunk 1 above)
Comment (aschobba) [reply doc/source/kaev_niederlausitz.md]: fixed

File: doc/source/kaev_niederlausitz.md (hunk 2 above)
Comment (aschobba) [reply doc/source/kaev_niederlausitz.md]: changed

**Review by aschobba — COMMENTED** [review]
updated README

**Review by aschobba — COMMENTED** [review]
typo fix

**Pull request merged**